packages/backend/common/models/club.js

```diff
@@ -8,11 +8,15 @@ const ClubSchema = Joi.object({
 
   name: Joi.string().required(),
 
-  description: Joi.string(),
+  description: Joi.string().required(),
 
-  officers: Joi.array().items(UserSchema),
+  officers: Joi.array().items(UserSchema).min(1),
```

Comment:
I think we should have the items as the User's id which is a String instead of a User object/schema

Reply:
Yes, I think that’s a good idea